civicrm.views_execution.inc

```diff
@@ -0,0 +1,32 @@
+<?php
+
+use Drupal\views\ViewExecutable;
+use Drupal\views\Plugin\views\query\QueryPluginBase;
+
+/**
+ * Implements hook_views_query_alter().
+ */
+function civicrm_views_query_alter(ViewExecutable $view, QueryPluginBase $query) {
+  \Drupal::service('civicrm')->initialize();
+
+  // check if we are in multilingual mode, otherwise return
+  // TODO: should be a simple call - is_multiligual ?
+  $domain = new CRM_Core_DAO_Domain();
```

Comment:
@mlutfy does this need a \ at the start to say we should be in the root namespace?

Reply:
I don't think it's required? It seems to work in both cases, but grepping around, I don't see `new \CRM`. The only place I see it, is under `civicrm-core/Civi/`.

Correct me if I'm wrong, but googling around, people seem to suggest avoiding it if not required, for code readability. It should only be used when there is a risk of a namespace clash between the local class and global scope.

(sorry for the slow response, had not seen the email notification on this thread)